discord/raw_models.py

```diff
@@ -250,6 +254,10 @@ class RawReactionClearEvent(_RawReprMixin):
     def __init__(self, data: ReactionClearEvent) -> None:
         self.message_id: int = int(data["message_id"])
         self.channel_id: int = int(data["channel_id"])
+        # For some reason this event has `burst` even though it makes no sense
```

Comment:
That's because it includes whether the reaction was a burst.
This likely won't change.

Reply:
But this is the full clear endpoint, which clears *all* reactions from the message. A single bool here wouldn't tell you anything useful.

Reply:
i believe you can't clear both normal and burst reactions at the same time, not sure tho.
We have to wait for the docs